src/tests/functional/plugin/shared/include/behavior/ov_plugin/core_integration.hpp

```diff
@@ -994,6 +995,27 @@ TEST_P(OVClassLoadNetworkTest, LoadNetworkWithBigDeviceIDThrows) {
     }
 }
 
+TEST_P(OVClassLoadNetworkWithAutoBatchingTest, LoadNetworkWithAutoBatchingNoThrows) {
+    ov::Core ie = createCoreWithTemplate();
+    ov::CompiledModel model;
+
+    // Disable Auto Batching
+    OV_ASSERT_NO_THROW(model = ie.compile_model(actualNetwork,
+                                                deviceName,
+                                                ov::hint::performance_mode(ov::hint::PerformanceMode::THROUGHPUT),
+                                                ov::hint::allow_auto_batching(false)));
+    ASSERT_THROW(model.get_property(ov::hint::auto_batching_device_config), ov::Exception);
```

Comment:
only the AUTo keeps the value of the allow_auto_batching and allows to query that.
Again, this is conventional test, nothing specific for this device or properties

so better to use instantiation for AUTO and the ov::hint::auto_batching_device_config